### DISCUSSION TITLE
My Account is Flagged

### DISCUSSION THREAD

**Discussion by PriyankaGan** [community/community#102039] [New to GitHub]
### Select Topic Area

Question

### Body

Hello Team , 

I was not aware that i should not create multiple user accounts as in the scenario was i created a wrong type of account, so i told to delete the account and they deleted the account. i tried to recreate the account and the same features appeared and i am unable to invite collaborators. Need help in creating a new account and reviewing the old accounts and disabling
![Screenshot 2024-01-24 115712](https://github.com/community/community/assets/157538614/d499f4f0-8317-40c6-a5c0-cfe6ad6c2ffb)
![Screenshot 2024-01-25 135148](https://github.com/community/community/assets/157538614/867a0390-50a0-408c-9e44-0b6169294f77)

How can i fix this trying to reach out git hub support they dont reply at all

**Comment by K-O-V-I-D** [comment]
Did you get any solution.!
